.github/workflows/release_plugin.yml

```diff
@@ -65,6 +65,15 @@ jobs:
         if: steps.semver_parser.outputs.prerelease == ''
         run: 'echo "{ \"latest\": \"${{github.ref_name}}\" }" > ./sites/versions/v2/${{steps.split.outputs.plugin_type}}-${{steps.split.outputs.plugin_name}}.json'
 
+      - name: Update version in example config files
```

Comment:
If this get a bit too much to maintain, maybe we can use the following approach:
https://github.com/cloudquery/.github/blob/ac27d588a9a9d492b7a3a2054638119d7c1df167/.github/renovate-terraform.json5#L19

We could move that config up to the default preset:
https://github.com/cloudquery/.github/blob/ac27d588a9a9d492b7a3a2054638119d7c1df167/.github/renovate-default.json5

Downside that we run renovate on a 1 hour schedule so it won't get updated immediately (we can change the schedule)

Reply:
Yeah, or maybe we should fork gha-find-replace and add the ability to use multiple glob matches. If we did that, we could collapse all three these steps into one.

Reply:
Okay no, it would still be 2 steps. But we could further widen the options to get it down to one.

Reply:
I think we might need do a follow up on this approach as some PRs are getting blocked due to conflicts:
https://github.com/cloudquery/cloudquery/pull/3052:
![image](https://user-images.githubusercontent.com/26760571/196727904-8ac41241-1fd8-4dfb-b79f-098b535edd72.png)